extensions/amp-social-share/0.1/amp-social-share.js

```diff
@@ -128,8 +128,12 @@ class AmpSocialShare extends AMP.BaseElement {
       const isMailTo = protocol === 'mailto:';
       const isSms = protocol === 'sms:';
       const isIosSafari = this.platform_.isIos() && this.platform_.isSafari();
+      const isIos13Webkit =
```

Comment:
This series of booleans seems more convoluted than necessary. It essentially boils down to if (Safari or iOS version > 13), use `_top` instead of `_blank`. 

Should we just combine the Safari and Webkit check? Maybe this can read better as: 

```
// mailto:, sms: protocols breaks when opened in _blank on Safari or iOS 13
const isSafariOrIos13 = this.platform_.isIoS() && this.platform_getIosMajorVersion() >= 13 || this.platform_.isSafari(); 
if (isSafariOrIos13 && (isMailTo || isSms)) {
this.target_ = '_top'; 
} else {
this.target_ = this.element.hasAttribute('data-target') || 'blank'; 
}
```

Reply:
I'm waiting for someone to confirm if this is existing on iOS < 13 as well, if so it'll be a less complicated check (all of iOS vs not iOS)

Reply:
@kristoferbaxter any luck with that?

Reply:
Verified that this is a problem on `iOS 12.4`. I think we can simplify this check to all webkit not Safari?